sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/SchedulerConfig.java

```diff
@@ -422,6 +427,19 @@ public TokenProvider getDcosAuthTokenProvider() throws IOException {
     }
   }
 
+  private JSONObject loadFileOrEnvSecret() throws IOException {
+    String content = envStore.getRequired(SIDECHANNEL_AUTH_ENV);
+    JSONObject serviceAccountObject;
+    if (Files.isRegularFile(Paths.get(content))) {
+      LOGGER.info("Reading file {} to load secrets", content);
+      serviceAccountObject = new JSONObject(IOUtils.toString(new FileInputStream(content)));
+    } else {
+      LOGGER.info("{} is not a valid path. Parsing as JSON content.", SIDECHANNEL_AUTH_ENV);
```

Comment:
I don't think that the `is not a valid path` is required here. What about:
```suggestion
      LOGGER.info("Reading service account information from {}.", SIDECHANNEL_AUTH_ENV);
```
As this does not give any information as to the nature of the data.